code-challenges.test.js

```diff
@@ -25,11 +46,30 @@ const secretCodeWord3 = "Eccentric"
 
 // b) Create the function that makes the test pass.
 
+const codedMessage = (x) => {
+  return x.replaceAll('a','4').replaceAll(/e|E/gi,'3').replaceAll('i','1').replaceAll('o','0');
+}
+
```

Comment:
Since you used regex to solve this problem it would be a good idea to add the process notes to your pseudo code.